src/components/SliderCard/SliderCard.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+import img from '../../img/img.png';
+import React from 'react';
+import Image from 'next/image';
+import './scss/SliderCard.scss';
+
+export const SliderCard: React.FC = () => {
+  return (
+    <div className="card">
+      <Image src={img} alt="img" />
```

Comment:
Alt should be more descriptive and contains dynamic image description. I know, for now everything is hard coded but still.